appinventor/docs/markdown/reference/blocks/math.md

```diff
@@ -391,4 +391,4 @@ Example:
 |---------|---------|---------|
 |         |       6 | 0 1 1 0 |
 |         |       3 | 0 0 1 1 |
-| Result: |       5 | 0 1 0 1 |
+| Result: |       5 | 0 1 0 1 |
```

Comment:
The newline shouldn't have been removed from the end of the file.

Reply:
@ewpatton are you referencing 394th line?

Reply:
@devarsh-mavani-19 Yes, I don't see why you've changed this line, but the result is that the file does not end with a newline.

Reply:
@ewpatton as you can see there are actually 2 commits. That line was changed accidently as i opened the file in VSCode. So i made another commit where i again changed the line as it is in the original version. and in the original version i don't see any new line after that. so should i add another commit adding a new line?

Reply:
![Screenshot from 2022-01-29 11-34-20](https://user-images.githubusercontent.com/55136047/151649751-3068e8b9-cbe9-48e6-809b-5ea81920d30c.png)
I have attached this image showing line count in original file

Reply:
@ewpatton I added a new line, then squashed recent updates and force pushed is that correct now?